buildpack.md

```diff
@@ -225,6 +230,24 @@ The lifecycle:
 
 Layers marked `launch = false`, `build = false`, and `cache = false` behave like temporary directories, available only to the authoring buildpack, that exist for the duration of a single build.
 
+## Exec.d Interface
```

Comment:
Could we move this next to the other table? That way you can get the whole interface summary at a glance.

Side note: worth creating a section like this for profile.d as well

Reply:
Right now it is a sibling of `Buildpack Interface` and `App Interface`. Are you suggesting that it should be a child of `Buildpack Interface`?